case-lib/lib.sh

```diff
@@ -47,11 +47,13 @@ func_lib_setup_kernel_last_line()
 SOF_LOG_COLLECT=0
 func_lib_start_log_collect()
 {
-    local is_etrace=${1:-0} ldcFile
-    ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || {
-        >&2 dlogw "sof-dump-status.py -p to query platform failed"
-        return
-    }
+    local is_etrace=${1:-0} ldcFile="$SOFLDC"
+    if [[ ! -e "$SOFLDC" ]]; then
```

Comment:
```shell
if [ -z "$SOFLDC" ]; then
```

If I misconfigure SOFLDC by mistake, then I want the logger to fail. I don't want something else to try to correct my mistake without telling me.

Reply:
updated